seeds/zeroex/celo/_schema.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+
+seeds:
+  - name: zeroex_celo_api_fills_sample
+    config:
+      tags: dunesql
+      column_types:
+        # TODO: Add column types here
```

Comment:
Hi @RantumBits,
please define the column types here.
You may move this schema file into `seeds/zeroex/` if you want.